roles/galaxy.createuser/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+#Use bioblend to create a Galaxy admin user
+- name: Create/invoke script virtualenv for create galaxy admin
+  pip: name={{ item }} virtualenv={{ galaxy_venv_dir }} virtualenv_command="{{ pip_virtualenv_command | default( 'virtualenv' ) }}"
+  with_items:
+    - pyyaml
+    - bioblend
+  sudo: yes
+  sudo_user: "{{ galaxy_user_name }}"
+
+- name: Place the data manager script
+  copy: src=create_galaxy_user.py dest={{ galaxy_tools_base_dir }}/
+
+- name: Create Galaxy admin user
```

Comment:
If you call this role createuser then you should be able to generate non-admin users, and I think it owuld be nice if the role manages the admin_users in galaxy's config file